package.json

```diff
@@ -11,13 +11,13 @@
   "license": "SSPL",
   "scripts": {
     "check": "lerna run check --stream",
-    "check-evergreen": "lerna run check --concurrency 1 --scope mongodb-compass",
+    "check-evergreen": "lerna run check",
```

Comment:
Check works nicely without concurrency limited (test still needs one, unless until we make sure that tests are not using same mongodb runner ports etc.)